checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Vigenere(Test):
```

Comment:
@brianyu28, to avoid new nomenclature, should we perhaps just call our class `check50.TestCase`? Or something like `Suite`, as in a "suite of checks"?